data/mods/blazemod/recipes/diamond_recipes.json

```diff
@@ -0,0 +1,81 @@
+[
```

Comment:
There were only a few recipes in `blaze_test.json`, but I felt it prudent to separate them out nevertheless immediately. Unchanged.